docs/plugins/deezer.rst

```diff
@@ -0,0 +1,38 @@
+Deezer Plugin
+==============
+
+The ``deezer`` plugin provides metadata matches for the importer using the
+`Deezer`_ `Album`_ and `Track`_ APIs.
+
+.. _Deezer: https://www.deezer.com
+.. _Album: https://developers.deezer.com/api/album
+.. _Track: https://developers.deezer.com/api/track
+
+Why Use This Plugin?
+--------------------
+
+* You're a Beets user.
+* You want to autotag music with metadata from the Deezer API.
```

Comment:
This section doesn't seem to add too much—both of these points are pretty well covered by your introductory sentence above. :smiley: